apps/ai/.env.example

```diff
@@ -0,0 +1,15 @@
+SENDER_EMAIL=
+
+BACKEND_URL=http://localhost:3002/api
+
+APP_ID=cal-ai
+APP_URL=http://localhost:3000/apps/cal-ai
+
+OPENAI_API_KEY=
+RESEND_API_KEY=
```

Comment:
i wonder, should we use sendgrid here?

Reply:
Any particular reason to not use nodemailer with existing env vars like `EMAIL_FROM`?

Reply:
Migrating to nodemailer.

As for `EMAIL_FROM`, the only concern would be replying `notifications@cal.com` (or the actual value) after the user emailed `hi@cal.ai`.

Come to think of it, this should be dynamic, allowing emails to `*@cal.ai`.